### PR TITLE
Added golangci-lint checker with default settings

### PR DESCRIPTION
**Pull request by Josca**
- added automatic checking
- fixed lint hints

### CODE REVIEW

File: base/database/setup.go [hunk 1]
```diff
@@ -83,16 +83,6 @@ func loadEnvPostgreSQLConfig() *PostgreSQLConfig {
 	return &config
 }
 
-// create config string from additional timeout params
-func timeoutParams(dbConfig *PostgreSQLConfig) string {
```
Comment (beav) [inline base/database/setup.go]: was this unused?

**Comment by beav** [comment]
@Josca can you check if the linting was executed in the PR tests? i didn't see it but its very possible I missed it when scrolling through.

File: base/database/setup.go (hunk 1 above)
Comment (Josca) [reply base/database/setup.go]: yes

**Comment by Josca** [comment]
Yes, it was only visible when code analysis failed. It was not ideal. I've added simple [message](https://github.com/RedHatInsights/patchman-engine/pull/64/checks?check_run_id=392736900#step:3:2173) after analysis finishing.

**Pull request merged**